src/backup/backup.c

```diff
@@ -296,6 +255,7 @@ backup_error_t backup_create(uint32_t backup_create_timestamp, uint32_t seed_bir
         // BACKUP_ERR_CHECK.
         // The caller could try again.
         uint8_t read_content[SD_MAX_FILE_SIZE];
+        CLEANUP_BACKUP_BYTES(read_content);
```

Comment:
How about moving this and `_cleanup_backup_bytes` to backup_common.h as well, since it already contains similar `backup_cleanup_backup` and `backup_cleanup_backup_data`?

Reply:
No need, it is only used here, so a static function suffices.